tests/create_sequence_data.py

```diff
@@ -63,11 +63,13 @@ def create_sequence_data(
         if shifts_per_signal:
             shifts = np.random.rand(n_signals) / meta["sampling_rate"] * 0.9
             np.save(SEQUENCE_ROOT / "meta" / "phase_shifts.npy", shifts)
+        else:
+            shifts = np.zeros(n_signals)
 
         with open(SEQUENCE_ROOT / "meta.yml", "w") as f:
             yaml.dump(meta, f)
 
-        yield timestamps, data, shifts if shifts_per_signal else None
+        yield timestamps, data, shifts
```

Comment:
@reneburghardt why do we need to return zeros for the phase shifts now?

Reply:
Whoops, that remained there accidentally from a change I didn't do in the end. I reverted it and pushed :)